docusaurus.config.js

```diff
@@ -66,6 +66,17 @@ const config = {
         siteUrl: "https://docs.flathub.org/",
       },
     ],
+    [
+      require.resolve("@docusaurus/plugin-client-redirects"),
+      {
+        redirects: [
+          {
+            to: "/docs/for-app-authors/maintenance",
```

Comment:
I would prefer to have from first and then to, but doesn't matter and I will probably just change it later :)